cockroachcloud/security-overview.md

```diff
@@ -5,93 +5,158 @@ toc: true
- </tr>
- <tr>
-   <td rowspan="2" ><a href="managing-access.html">User Authorization</a></td>
-   <td>✓</td>
-   <td>✓</td>
-   <td>Users and privileges</td>
- </tr>
- <tr>
-  <td>✓</td>
-  <td>✓</td>
-  <td>Role-based access control</td>
- </tr>
- <tr>
-   <td rowspan="3"><a href="network-authorization.html">Network Authorization</a></td>
-   <td>&nbsp;</td>
-   <td>✓</td>
-   <td>IP allowlisting</td>
- </tr>
- <tr>
-  <td>&nbsp;</td>
-  <td>✓</td>
-  <td><a href="private-clusters.html">Private Clusters (Preview)</a></a> whose clusters nodes have no public IP addresses </td>
- </tr>
- <tr>
-  <td>&nbsp;</td>
-  <td>✓</td>
-  <td><a href="network-authorization.html">VPC Peering</a> for GCP clusters and <a href="network-authorization.html">AWS PrivateLink</a> for AWS clusters </td>
- </tr>
- <tr>
-   <td><a href="../{{site.current_cloud_version}}/cluster-api.html">Cluster API</a></td>
-   <td>&nbsp;</td>
-   <td>✓</td>
-   <td>HTTP API access using login tokens</td>
- </tr>
+    <th width="120">Security Domain</th>
+    <th>CockroachDB Serverless</th>
+    <th>CockroachDB Dedicated</th>
+    <th>Feature</th>
+  </tr>
+  <tr>
+    <td rowspan="9"><a href="authentication.html">Authentication</a></td>
+    <td>✓</td>
+    <td>✓</td>
+    <td>Inter-node and node identity authentication using TLS 1.3</td>
+  </tr>
+  <tr>
+    <td>✓</td>
+    <td>✓</td>
+    <td>Client identity authentication using a username and password</td>
+  </tr>
+  <tr>
+    <td>✓</td>
+    <td>✓</td>
+    <td><a href="../{{site.versions['stable']}}/security-reference/scram-authentication.html">SASL/SCRAM-SHA-256 secure password-based authentication</a></td>
+  </tr>
+  <tr>
+    <td>&nbsp;</td>
+    <td>✓</td>
+    <td>Web console authentication with third-party <a href="../{{site.versions['stable']}}/sso-db-console.html">Single Sign On (SSO)</a> using <a href="https://openid.net/connect/">OpenID Connect OIDC</a> or <a href="https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language">SAML</a></td>
+  </tr>
+  <tr>
+    <td>✓</td>
+    <td>✓</td>
+    <td>SQL Client authentication with <a href="cloud-sso-sql.html">Cluster SSO</a> using CockroachDB Cloud as identity provider</td>
+  </tr>
+  <tr>
+    <td>&nbsp;</td>
+    <td>✓</td>
+    <td>SQL Client authentication with <a href="../{{site.versions['stable']}}/sso-sql.html">Cluster SSO</a> using customer-managed identity providers</td>
+  </tr>
+  <tr>
+    <td>&nbsp;</td>
+    <td>✓</td>
+    <td>Client identity authentication using <a href="client-certs-dedicated.html">PKI certificates</a></td>
+  </tr>
+  <tr>
+    <td>&nbsp;</td>
+    <td>✓</td>
+    <td><a href="../{{site.versions['stable']}}/manage-certs-revoke-ocsp.html">OCSP</a> certificate revocation protocol</td>
+  </tr>
+  <tr>
+    <td>&nbsp;</td>
+    <td>✓</td>
+    <td><a href="../{{site.current_cloud_version}}/cluster-api.html">Cluster API</a> access using login tokens</td>
```

Comment:
If this is referring to CRDB's cluster API, we should remove it from this cloud specific page.

Reply:
ok